test/normalize.js

```diff
@@ -22,8 +22,12 @@ describe('nemo-view @normalize@ module', function () {
         type: 'xpath',
         locator: '/x/y/z:[abc]'
       },
-      'output': nemo.wd.By.xpath('/x/y/z:[abc]')
-    }, {
+      'output': {
+        type: 'xpath',
+        locator: '/x/y/z:[abc]'
+      }
```

Comment:
since this is equivalent to an object of `nemo.wd.By` we don't normalize/recreate, so `input` is equal to `output` here in this case.